cypress/integration/login/signup_spec.js

```diff
@@ -67,12 +77,12 @@ describe('Signup Email page', () => {
         cy.get('#copyright').should('contain', '© 2015-');
         cy.get('#copyright').should('contain', 'Mattermost, Inc.');
         cy.get('#about_link').should('contain', 'About');
-        cy.get('#about_link').should('have.attr', 'href', 'https://about.mattermost.com/default-about/');
+        cy.get('#about_link').should('have.attr', 'href', config.SupportSettings.AboutLink);
         cy.get('#privacy_link').should('contain', 'Privacy');
-        cy.get('#privacy_link').should('have.attr', 'href', 'https://about.mattermost.com/default-privacy-policy/');
+        cy.get('#privacy_link').should('have.attr', 'href', config.SupportSettings.PrivacyPolicyLink);
         cy.get('#terms_link').should('contain', 'Terms');
-        cy.get('#terms_link').should('have.attr', 'href', 'https://about.mattermost.com/default-terms/');
+        cy.get('#terms_link').should('have.attr', 'href', config.SupportSettings.TermsOfServiceLink);
         cy.get('#help_link').should('contain', 'Help');
-        cy.get('#help_link').should('have.attr', 'href', 'https://about.mattermost.com/default-help/');
+        cy.get('#help_link').should('have.attr', 'href', config.SupportSettings.HelpLink);
```

Comment:
This is failing since cypress server has different config for `SiteName` and so, it's safe to use data from config instead of hard-coded values.